sensor_msgs/Image class_segment_img
sensor_msgs/Image instance_segment_img
string[] class_names
uint16[] class_ids
